lunatrace/bsl/hasura/migrations/lunatrace/1669922910149_add_root_to_folder_options/up.sql

```diff
@@ -0,0 +1,85 @@
+ALTER TABLE public.project_folder_settings
+    ADD COLUMN root BOOLEAN NOT NULL DEFAULT FALSE;
+
+-- Only one root setting per project
+CREATE UNIQUE INDEX one_root_folder_setting_only ON public.project_folder_settings (project_id)
+    WHERE root = TRUE;
+
+ALTER TABLE public.cvss_environmental_adjustment
+    ADD COLUMN group_name TEXT;
+ALTER TABLE public.cvss_environmental_adjustment
+    ALTER COLUMN group_name SET NOT NULL;
+
+--  internet exposure
+--  TODO: when implementing this in the backend, make sure these internet settings can only LOWER a vulnerabilities value and not raise it, since these are overrides
+INSERT INTO public.cvss_environmental_adjustment (group_name, name, attack_vector)
+VALUES ('internet', 'internet', 'N'),
```

Comment:
should probably give these known uuids because they will be referenced in code (i think)

Reply:
We discussed that.  It definitely occured to me. We landed on using the names as the way to identify them in code. 

The reasons we went with that 
1.) We want to have this feature be easily expandable with user-entered values in the future. Those obviously can't be based around known IDs and need to be more dynamic.
2.) Manually creating special known UUIDs feels so weird. Mainly, it's 1 though